examples/creates-nested-ampcontext-creative.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
```

Comment:
what do you think if we introduce another endpoint for this:
`/nameframe/ampcontext-creative.html`

so we can have nested nameframes like this:
`/nameframe/nameframe/ampcontext-creative.html`